.agent/rules/goldskypipelines.md

```
@@ -0,0 +1,261 @@
+# Operating pipelines
+
+> Guide to common pipeline operations
+
+### Deploying a pipeline
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)